src/app/pages/waiting-room-to-car/waiting-room-to-car.analytics.effects.ts

```diff
@@ -470,12 +483,259 @@ export class WaitingRoomToCarAnalyticsEffects {
       filter(([, , , practiceMode]) =>
         !practiceMode ? true : this.appConfigProvider.getAppConfig()?.journal?.enablePracticeModeAnalytics
       ),
-      switchMap(([, tests, motStatus]: [ReturnType<typeof MotStatusChanged>, TestsModel, string, boolean]) => {
+      switchMap(([, tests, motStatus]: [ReturnType<typeof MotStatusChanged>, TestsModel, MotStatusCodes, boolean]) => {
         // GA4 Analytics
         this.analytics.logGAEvent(
           analyticsEventTypePrefix(GoogleAnalyticsEvents.MOT_CHECK, tests),
           GoogleAnalyticsEventsTitles.MOT_STATUS,
-          motStatus
+          motStatus == MotStatusCodes.AGE_EXEMPTION ? MotStatusCodes.NO_DETAILS : motStatus
+        );
+        return of(AnalyticRecorded());
+      })
+    )
+  );
+
+  motCallAborted$ = createEffect(() =>
+    this.actions$.pipe(
+      ofType(MotCallAborted),
+      concatMap((action) =>
+        of(action).pipe(
+          withLatestFrom(this.store$.pipe(select(getTests)), this.store$.pipe(select(getTests), select(isPracticeMode)))
+        )
+      ),
+      filter(([, , practiceMode]) =>
+        !practiceMode ? true : this.appConfigProvider.getAppConfig()?.journal?.enablePracticeModeAnalytics
+      ),
+      switchMap(([{ method }, tests]: [ReturnType<typeof MotCallAborted>, TestsModel, boolean]) => {
+        let abortMethod = GoogleAnalyticsEventsValues.UNKNOWN;
+
+        switch (method) {
```

Comment:
a switch without a default is bad practice.  suggest that a setting to GoogleAnalyticsEventsValues.UNKNOWN is handled via a default

switch (method) {
  case MOTAbortedMethod.END_TEST:
    abortMethod = GoogleAnalyticsEventsValues.END_TEST_SELECTED;
    break;
  case MOTAbortedMethod.VRN_CHANGED:
    abortMethod = GoogleAnalyticsEventsValues.VRN_EDITED;
    break;
  case MOTAbortedMethod.NAVIGATION:
    abortMethod = GoogleAnalyticsEventsValues.NAVIGATION;
    break;
  default:
    abortMethod = GoogleAnalyticsEventsValues.UNKNOWN;
}